### ISSUE TITLE
[BITV] 9.3.3.1/3.1 - Modal dialog "Choose target folder" - The error message "File name cannot be empty" is not programmatically associated with the input field and is also misleading, as the error message refers to the file name rather than the directory. Since the error message is not part of the label, it should be linked to the relevant field using the aria-describedby attribute. (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#37019]
- [ ] Fix
- [ ] Backport

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-c3b56e67ab-v3-n1

</details>

**Comment by susnux** [comment]
This is already fixed with the vue based file picker as the error message is now a native validity on the input element.
